# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/rene/qcad/gcc34.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./qcadlib/src/filters/rs_filtercxf.h.orig	2004-09-11 05:17:51.961336120 +0200
+++ ./qcadlib/src/filters/rs_filtercxf.h	2004-09-11 05:14:22.931113544 +0200
@@ -28,6 +28,8 @@
 #ifndef RS_FILTERCXF_H
 #define RS_FILTERCXF_H
 
+#undef min
+#undef max
 #include <fstream>
 
 #include "rs_filterinterface.h"
--- ./qcadlib/src/filters/rs_filterdxf.h.orig	2004-09-11 05:17:36.443695160 +0200
+++ ./qcadlib/src/filters/rs_filterdxf.h	2004-09-11 05:13:50.997968120 +0200
@@ -44,6 +44,8 @@
 #include "rs_image.h"
 
 #include "dl_creationinterface.h"
+#undef min
+#undef max
 #include "dl_dxf.h"
 
 
--- ./qcad/src/qc_applicationwindow.cpp.orig	2004-09-11 05:39:43.001027792 +0200
+++ ./qcad/src/qc_applicationwindow.cpp	2004-09-11 05:40:19.329505024 +0200
@@ -26,6 +26,8 @@
 
 #include "qc_applicationwindow.h"
 
+#undef min
+#undef max
 #include <fstream>
 
 #include <qaccel.h>
